mentat/terminal/loading.py

```diff
@@ -0,0 +1,37 @@
+from tqdm import tqdm
```

Comment:
Would it be best to use PromptToolkit's progress bar? https://python-prompt-toolkit.readthedocs.io/en/master/pages/progress_bars.html I have no experience with this one, but it might mesh better with prompt toolkit and since we already use prompt toolkit I thought we might want to think about it

Reply:
I wasn't aware of that! Looks like a lot less functionality than tqdm - including some key things I'm using like updating the label, or using it without a context manager. I think it makes sense to stick with tqdm for now.